packages/transaction-controller/src/TransactionControllerIntegration.test.ts

```diff
@@ -1772,6 +1772,58 @@ describe('TransactionController Integration', () => {
       });
     });
 
+    describe('when a transaction is added with a networkClientId that is removed during the lifecycle', () => {
+      it('should not advanced to unapproved if there is no fallback provider on the same chainId', () => {
```

Comment:
```suggestion
      it.todo('should not advanced to unapproved if there is no fallback provider on the same chainId', () => {
```

wanna mark these all with `.todo` for now?